### DISCUSSION TITLE
citing DANDI?

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#68] [Q&A]
the Dandisets have their own citations, but how do we cite the DANDI archive itself? Like in a grant if we are proposing to deposit data on DANDI or integrate with the DANDI Hub.

**Comment by yarikoptic** [comment]
For dandi-cli we have citations through zenodo: https://zenodo.org/record/6412673#.Yk9I0VvMJUE . I wonder if we should enable (and tune up) one for `dandi-archive` repo as well?

In the long(er) run we should finally write some DANDI paper to provide a canonical citation. Should we start @satra?

**Reply by satra** [reply]
we should do a zenodo doi for the dandi-archive repo as well. i did start a paper. @bendichter - you could cite the nwb ecosystem preprint for now. many places are fine with simply citing the website (dandiarchive.org).
